processing_TEMPLATE.yaml

```diff
@@ -62,6 +62,6 @@ audio:
     analysis_workers: 2
 
     # The command will be called to perform analysis on audio recordings (e.g. Cacophony Index, speech detection) using AI models
-    analysis_command: 'docker run --rm -v {folder}:/io cacophonyproject/audio-analysis:{tag} /io/"{basename}"'
+    analysis_command: 'docker run --rm -v {folder}:/io cacophonyproject/audio-analysis:{tag} /io/"{basename}"  --morepork-model none --analyse-tracks {analyse_tracks}'
```

Comment:
Is cli option taking in a boolean value? Probably should be cli flag without the value if so.

Reply:
nah it could be a path to a morepork model

Reply:
but default is to use it so need the None option